server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/pipeline_configs/_accordion.scss

```diff
@@ -14,18 +14,29 @@
  * limitations under the License.
  */
 
+$accordion-backg: #CFD8DC;
```

Comment:
It's a bit inconsistent with other variables, part of which are declared in `go-variables.scss`, ideally we'd want one place to configure it, `_settings.scss` is preferred, since it'll survive a foundation upgrade, these changes may not.

If at all possible, tweaks should go into `_settings.scss` before overriding it anywhere else.